tesseract/tesseract_planning/tesseract_motion_planners/examples/raster_example.cpp

```diff
@@ -61,6 +65,16 @@ int main(int /*argc*/, char** /*argv*/)
   boost::filesystem::path srdf_path(std::string(TESSERACT_SUPPORT_DIR) + "/urdf/abb_irb2400.srdf");
   tesseract->init(urdf_path, srdf_path, locator);
 
+  // Dynamically load ignition visualizer if exist
+  std::string library_path = "/home/larmstrong/catkin_ws/trajopt_only_ws/devel/lib/"
+                             "libtesseract_ignition_visualization_plugin.so";
+  std::string class_name = "tesseract_ignition::TesseractIgnitionVisualization";
+  tesseract_visualization::VisualizationLoader loader(library_path, class_name);
```

Comment:
Same thing regarding hardcoded filepath

Reply:
Removed, these were for testing and should have been removed. The default constructor will initialize with paths to the snap location.